sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionInfo.java

```diff
@@ -26,6 +26,10 @@
     private String name;
     private String extended;
     private String db;
+    private String arguments;
```

Comment:
There aren't many `ExpressionInfo` objects in memory right? adding more info to this bean doesn't have any meaningful performance implications, I presume. I suppose it's just breaking down the existing info further.

I also presume this is considered an internal API so it's OK to change the constructor. You could even retain the one constructor that is removed, just in case.